### ISSUE TITLE
[Suggestion] Enable common Error Handlers with HTML pages

### ISSUE THREAD

**Issue by gabbydgab** [samsonasik/ExpressiveRedirectHandler#4]
Needed: 
1. 401 Unauthorized -> redirect to login page

Nice to have:
1. 402 Payment Required -> redirect to checkout page
2. 503 Service Unavailable -> redirect to maintenance page

For discussion:
1. 403 Forbidden -> for registration/update credentials 
2. 440 Login Timeout -> for re-login

**Comment by samsonasik** [comment]
that seems ok as far as configurable, for example:

``` php
return [
     'expressive-redirect-handler' => [
             //...
             'header_handler' => [
                   401 => '/login',
            ],
     ],
];
```

do you want to create a PR for it ?

**Comment by gabbydgab** [comment]
Would loved to but unfortunately my time ATM is limited. Will look into it probably next few weeks.

@acelaya  also have pointers on how it should be done, depending on the implementation. But let say it will be applicable for both API and WEB apps, then we can set boundaries on the limitation of its feature.

My point is I just want to emphasize how to leverage the ['middleware_pipeline']['error']['middleware'] in the configuration as a "defacto" expressive feature since we can use it with events also.

Looking forward to have this feature. :smile:

**Comment by samsonasik** [comment]
@gabbydgab I've created PR #5 for it ;), would you mind to review ? Thank you.

**Comment by samsonasik** [comment]
implemented at #5 and released 0.2.0 for it.
